projects/ion/src/lib/step/step.component.html

```diff
@@ -0,0 +1,57 @@
+<section class="steps-container" data-testid="ion-steps">
+  <div
+    [class]="'step ' + step.status"
+    [class.clickable]="
+      clickable &&
+      !disabled &&
+      step.status !== 'selected' &&
+      step.status !== 'error'
+    "
+    [class.disabled]="disabled"
+    [attr.data-testid]="'step-' + step.index + '-' + step.status"
+    *ngFor="let step of steps; let index = index"
+    (click)="goesTo(step.index)"
+  >
+    <div class="step-draw">
+      <div
+        class="line"
+        [class.bolded]="beforeStepIsChecked(index)"
+        *ngIf="lineVisibility('initial', step)"
```

Comment:
I believe you don't need the lineVisibility function. Try 
```suggestion
        *ngIf="step.index !== firstStep"
```
where, fristStep is a const with value 1 in ts. you can do something similar for next div and delete lineVisibility